allennlp/commands/bulk.py

```diff
@@ -2,19 +2,22 @@
 from contextlib import ExitStack
 import json
 import sys
-from typing import Sequence, Optional, IO
+from typing import Optional, IO
 
 from allennlp.service.servable import Servable, ServableCollection
 
+def add_bulk_subparser(parser: argparse._SubParsersAction) -> argparse.ArgumentParser:  # pylint: disable=protected-access
+    description = '''Run the specified model against a JSONL input file.'''
+    subparser = parser.add_parser(
+            'bulk', description=description, help='Run a model in bulk.')
+    subparser.add_argument('model', type=str, help='the name of the model to run')
```

Comment:
Oh, this isn't what I was expecting and is also not required, because if we are evaluating a model on a large amount of data, we will probably be using one of the `DatasetReaders`, which contain dataset paths in the parameter file. Everything required to run or evaluate one of the models should be contained within the parameter file.